velox/type/Tokenizer.h

```diff
@@ -51,6 +53,7 @@ class Tokenizer {
   const char UNICODE_CARET = '^';
 
   const std::string path_;
+  std::vector<char> regularChars_;
```

Comment:
I think `std::optional<std::vector>` is better. For no regularChars specified case, just check whether the optional has value and skip the unnecessary iteration finding if it doesn't.